src/components/checkbox/checkbox.ts

```diff
@@ -106,7 +107,9 @@ export class MdCheckbox implements ControlValueAccessor {
   @Input() name: string = null;
 
   /** Event emitted when the checkbox's `checked` value changes. */
-  @Output() change: EventEmitter<boolean> = new EventEmitter<boolean>();
+  @Output()
+  change: Observable<boolean>;
+  _change: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
shouldn't the `@Output()` be on the `EventEmitter`? (I guess as `@Output('change')`)